main/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,280 @@
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-11-18 10:32+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: admin.py:28
+msgid "No associated building"
+msgstr "Kein zugeordnetes Gebäude"
+
+#: admin.py:32
+msgid "link to the building"
+msgstr "Link zum Gebäude"
+
+#: admin.py:69
+msgid "link to the property"
+msgstr "Link zr Einheit"
+
+#: admin.py:72
+msgid "tenant administration"
+msgstr "Mieterverwaltung"
+
+#: admin.py:80
+msgid "tenant reminder list"
+msgstr "Mieter Erinnerungsliste"
+
+#: admin.py:81
+msgid "tenants reminder lists"
+msgstr "Mieter Erinnerungslisten"
+
+#: models.py:22 models.py:42 models.py:53 models.py:79 models.py:102
+#: models.py:172
+msgid "name"
+msgstr "Name"
+
+#: models.py:23 models.py:59 models.py:111
+msgid "notes"
+msgstr "Notizen"
+
+#: models.py:26
+msgid "building"
+msgstr "Gebäude"
+
+#: models.py:27
+msgid "buildings"
+msgstr "Gebäude"
+
+#: models.py:36
+msgid "number of properties"
+msgstr "Anzahl Einheiten"
+
+#: models.py:43 models.py:46 models.py:80 models.py:83 models.py:173
+#: models.py:176
+msgid "file"
+msgstr "datei"
+
+#: models.py:58
+msgid "address"
+msgstr "Adresse"
+
+#: models.py:61
+msgid "surface area"
+msgstr "Fläche"
+
+#: models.py:64
+msgid "number of rooms"
+msgstr "Anzahl Räume"
+
+#: models.py:68
+msgid "property"
+msgstr "Einheit"
+
+#: models.py:69
+msgid "properties"
+msgstr "Einheiten"
+
+#: models.py:84 models.py:177
+msgid "files"
+msgstr "dateien"
+
+#: models.py:93
+msgid "month expected. Please use first day of the month"
+msgstr ""
+
+#: models.py:104
+msgid "tenancy begin date"
+msgstr "Mietbeginn"
+
+#: models.py:106
+msgid "tenancy end date"
+msgstr "Mietende"
+
+#: models.py:108
+msgid "deposit"
+msgstr "kaution"
+
+#: models.py:110
+msgid "contact info"
+msgstr "Kontakt"
+
+#. Translators: This is the balance of the tenant's payments
+#: models.py:146 templates/main/cashflows.html:11
+#: templates/main/tenants.html:18
+msgid "balance"
+msgstr "Soll / Haben"
+
+#: models.py:162 templates/main/tenants.html:17
+msgid "tenant"
+msgstr "Mieter"
+
+#: models.py:163 templates/base.html:30
+msgid "tenants"
+msgstr "Mieter"
+
+#: models.py:185 models.py:225 models.py:243 templates/main/cashflows.html:9
+msgid "date"
+msgstr "Datum"
+
+#: models.py:186 models.py:223 models.py:241
+msgid "description"
+msgstr "Beschreibung"
+
+#: models.py:188
+msgid "mark as read"
+msgstr "Als gelesen markieren"
+
+#: models.py:191
+msgid "reminder"
+msgstr "Erinnerung"
+
+#: models.py:192 templates/main/tenants.html:37
+msgid "reminders"
+msgstr "Erinnerungen"
+
+#: models.py:201
+msgid "start date"
+msgstr "Startdatum"
+
+#: models.py:203
+msgid "end date"
+msgstr "Enddatum"
```

Comment:
In general, having an uppercase letter at the start of the word is a job for the template [exemple here](https://github.com/oadam/proprio/blob/master/main/templates/main/cashflows.html#L7) . When the untranslated string does contain un uppercase letter, that is an error. Please let's use lowercase everywhere